src/components/TryExtensionLinkDialog.test.tsx

```diff
@@ -0,0 +1,82 @@
+import { fireEvent, render, screen, waitFor } from "@testing-library/preact";
+import { h } from "preact";
+
+import { TryExtensionLinkDialog } from "./TryExtensionLinkDialog";
+
+const renderTryExtensionLinkDialog = ({
+  connectDisabled = false,
+  isConnected = true
```

Comment:
```suggestion
  isConnected = true,
```
i'd expect the linter to fail on this tbh

Reply:
The trailing commas option is set to none in prettier configuration so it just gets auto removed

Reply:
We can set trailing commas to `true` on a separate PR, I also prefer it on

Reply:
true